ticket_facts.view.lkml

```diff
@@ -0,0 +1,270 @@
+view: ticket_facts {
+  derived_table: {
+    sql: SELECT
+          audits.ticket_id,
+          tickets.status,
+          tickets.via__channel,
+          tickets.via__source__rel,
+          tickets.via__source__to__name,
+          groups.name,
+          metrics.group_stations AS number_of_groups_involved,
+          COUNT(DISTINCT audits.id) FILTER(WHERE audits.via__channel = 'voice' AND audits.via__source__from__name = 'Getaround' AND audits.via__source__rel = 'outbound')  AS number_outbound_calls,
+          COUNT(DISTINCT audits.id) FILTER(WHERE (audits.via__channel = 'voice' AND audits.via__source__to__name = 'Getaround' AND audits.via__source__rel = 'inbound')
+                                  OR (audits.via__channel = 'voice' AND audits.via__source__rel = 'voicemail'))  AS number_inbound_calls,
+          COUNT(DISTINCT audits.id) FILTER(WHERE (events.type = 'Comment' AND events.public = true AND audits.via__source__to__name = 'Getaround')
+                                  OR (events.type = 'Comment' AND events.public = true AND audits.via__channel = 'mobile_sdk')
+                                  OR (events.type = 'Comment' AND events.public = true AND audits.via__source__rel = 'follow_up'))  AS number_inbound_emails,
+          COUNT(DISTINCT audits.id) FILTER(WHERE events.type = 'Comment' AND events.public = true AND
+                                 (audits.via__source__to__name IS NULL OR audits.via__source__to__name <> 'Getaround'))  AS number_outbound_emails,
+          COUNT(DISTINCT audits.id) FILTER(WHERE events.type = 'Comment' AND events.public = false)  AS number_internal_comments,
+          COUNT(DISTINCT audits.id) FILTER(WHERE audits.via__source__rel = 'merge' and audits.via__source__from__ticket_id IS NULL) AS number_merged_tickets,
+          COUNT(DISTINCT audits.id) FILTER(WHERE audits.via__source__rel = 'merge' and audits.via__source__from__ticket_id IS NOT NULL)  AS is_ticket_merged,
```

Comment:
For this column `is_ticket_merged`, you can use [`EXISTS`](https://www.postgresql.org/docs/8.1/static/functions-subquery.html) to evaluate to a boolean type, rather than an integer.

If you change this, you need to change the SQL for dimension `is_ticket_merged` as well.